daal4py/sklearn/_utils.py

```diff
@@ -17,13 +17,16 @@
 import os
 import sys
 import warnings
+from typing import Any, Tuple
 
 import numpy as np
 from numpy.lib.recfunctions import require_fields
 from sklearn import __version__ as sklearn_version
 
 from daal4py import _get__daal_link_version__ as dv
 
+DaalVersionTuple = Tuple[int, str, int]
```

Comment:
i guess this have to be 4 elements 
Major - 2024
Minor - 0 
Patch - 1
Status - P 

Might be play with outputs/prints in case of this.

Reply:
For some reason the convention in `daal4py` is `(Major, Status, Minor+Patch)` - so only three elements

Reply:
> For some reason the convention in `daal4py` is `(Major, Status, Minor+Patch)` - so only three elements

ok, just to ensure that <= comparison would actually work. Like how Minor+Patch is done. I guess it should be ok, but a point to doublecheck

Reply:
I did some tests locally and they all worked as expected. And I didn't change any logic. I just updated the types and the comments to align with the code (which looks like it was changed some time without updating the comments).